package.json

```diff
@@ -120,10 +115,9 @@
 		}
 	],
 	"scripts": {
-		"vscode:prepublish": "export NODE_ENV=production && make frontend && webpack --mode production",
+		"vscode:prepublish": "export NODE_ENV=production && make frontend && make clean && make snippets && webpack --mode production",
```

Comment:
I assume I can also remove all the webpack stuff?

Reply:
Nope -- this webpack invocation is what actually compile the extension!